Code/Framework/AzToolsFramework/AzToolsFramework/UI/Outliner/EntityOutlinerTreeView.cpp

```diff
@@ -128,7 +130,8 @@ namespace AzToolsFramework
             setSelectionMode(QAbstractItemView::NoSelection);
 
             //treat this as a mouse pressed event to process everything but selection, but use the position data from the mousePress message
-            processQueuedMousePressedEvent(m_queuedMouseEvent);
+            QMouseEvent mouseEvent = ToMousePressEvent(m_queuedMouseEvent);
+            QAbstractItemView::mousePressEvent(&mouseEvent);
```

Comment:
need to route to call to the base class because QTreeView handles expanding fields.